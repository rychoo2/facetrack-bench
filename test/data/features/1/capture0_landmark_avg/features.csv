filename,raw_path,landmark_path,rel_target_x,rel_target_y,timestamp,rel_face_x,rel_face_y,rel_face_size_x,rel_face_size_y,rel_pose_x,rel_pose_y,rel_left_eye_x,rel_left_eye_y,rel_right_eye_x,rel_right_eye_y,rel_left_pupil_face_x,rel_left_pupil_face_y,rel_eye_distance_x,rel_eye_distance_y,rel_right_pupil_face_x,rel_right_pupil_face_y,rel_left_pupil_x,rel_left_pupil_y,rel_right_pupil_x,rel_right_pupil_y
images/20191123163439.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.9621337890625,0.17541714891975307,20191123163439000,0.471484375,0.46041666666666664,0.133203125,0.2361111111111111,0.5161290322580645,0.611764705882353,0.5249266862170088,-1.6147058823529412,-0.20821114369501467,-1.5382352941176471,0.9002932551319648,-1.5029411764705882,0.6011730205278593,-0.03529411764705882,0.13196480938416422,-1.438235294117647,0.46715328467153283,0.475,0.43283582089552236,0.4358974358974359
images/20191123163439.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.9621337890625,0.17541714891975307,20191123163439000,0.471484375,0.46041666666666664,0.133203125,0.2361111111111111,0.5161290322580645,0.611764705882353,0.5249266862170088,-1.6147058823529412,-0.20821114369501467,-1.5382352941176471,0.9002932551319648,-1.5029411764705882,0.6011730205278593,-0.03529411764705882,0.13196480938416422,-1.438235294117647,0.46715328467153283,0.475,0.43283582089552236,0.4358974358974359
images/20191123163438.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.7459716796875,0.2431785300925926,20191123163438000,0.47109375,0.46111111111111114,0.133203125,0.2361111111111111,0.5366568914956011,0.6029411764705882,0.5571847507331378,-1.6058823529411765,-0.17595307917888564,-1.535294117647059,0.9442815249266863,-1.5058823529411764,0.6041055718475073,-0.03235294117647059,0.17008797653958943,-1.4411764705882353,0.4925373134328358,0.4722222222222222,0.45038167938931295,0.4444444444444444
images/20191123163438.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.7459052191840277,0.24323157793209876,20191123163438000,0.47109375,0.46111111111111114,0.133203125,0.2361111111111111,0.5366568914956011,0.6029411764705882,0.5571847507331378,-1.6058823529411765,-0.17595307917888564,-1.535294117647059,0.9442815249266863,-1.5058823529411764,0.6041055718475073,-0.03235294117647059,0.17008797653958943,-1.4411764705882353,0.4925373134328358,0.4722222222222222,0.45038167938931295,0.4444444444444444
images/20191123163459.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.7609388563368056,0.9831090856481481,20191123163459000,0.452734375,0.02013888888888889,0.209375,0.37222222222222223,0.4962686567164179,0.6455223880597015,0.47201492537313433,0.2332089552238806,0.009328358208955223,0.27425373134328357,0.7033582089552238,0.27798507462686567,0.6529850746268657,-0.05970149253731343,0.21082089552238806,0.31902985074626866,0.4881889763779528,0.46153846153846156,0.4251968503937008,0.46153846153846156
images/20191123163459.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.7609388563368056,0.9831090856481481,20191123163459000,0.452734375,0.02013888888888889,0.209375,0.37222222222222223,0.4962686567164179,0.6455223880597015,0.47201492537313433,0.2332089552238806,0.009328358208955223,0.27425373134328357,0.7033582089552238,0.27798507462686567,0.6529850746268657,-0.05970149253731343,0.21082089552238806,0.31902985074626866,0.4881889763779528,0.46153846153846156,0.4251968503937008,0.46153846153846156
images/20191123163458.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.2961222330729167,0.9832947530864198,20191123163458000,0.4765625,0.025694444444444443,0.231640625,0.41180555555555554,0.5379426644182125,0.6188870151770658,0.48903878583473864,0.22765598650927488,0.03709949409780776,0.2580101180438449,,,0.5868465430016864,-0.050590219224283306,0.2596964586846543,0.31871838111298484,,,0.48175182481751827,0.5
images/20191123163458.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.29605712890625,0.9832368827160494,20191123163458000,0.4765625,0.025694444444444443,0.231640625,0.41180555555555554,0.5379426644182125,0.6188870151770658,0.48903878583473864,0.22765598650927488,0.03709949409780776,0.2580101180438449,,,0.5868465430016864,-0.050590219224283306,0.2596964586846543,0.31871838111298484,,,0.48175182481751827,0.5
images/20191123163503.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.06577284071180556,0.07994550540123457,20191123163503000,0.49296875,0.4618055555555556,0.133984375,0.23819444444444443,0.5160349854227405,0.5860058309037901,0.6647230320699709,-1.5714285714285714,-0.052478134110787174,-1.5772594752186588,1.0845481049562682,-1.5364431486880465,0.5714285714285714,0.01749271137026239,0.32653061224489793,-1.478134110787172,0.5950413223140496,0.15,0.4961832061068702,0.3953488372093023
images/20191123163502.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.06580539279513889,0.07994550540123457,20191123163502000,0.493359375,0.4618055555555556,0.13359375,0.2375,0.5087719298245614,0.5906432748538012,0.6637426900584795,-1.5994152046783625,-0.07309941520467836,-1.587719298245614,1.0789473684210527,-1.54093567251462,0.5847953216374269,0.014619883040935672,0.31871345029239767,-1.4707602339181287,0.5725806451612904,0.23255813953488372,0.5,0.46511627906976744
images/20191123163500.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.9681871202256944,0.9810860339506173,20191123163500000,0.455078125,0.019444444444444445,0.20546875,0.36527777777777776,0.5019011406844106,0.6482889733840305,0.47338403041825095,0.23954372623574144,0.009505703422053232,0.2737642585551331,0.7015209125475285,0.27756653992395436,0.6577946768060836,-0.058935361216730035,0.20342205323193915,0.3269961977186312,0.46153846153846156,0.43478260869565216,0.4015748031496063,0.5384615384615384
images/20191123163500.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.9680894639756944,0.9810860339506173,20191123163500000,0.455078125,0.019444444444444445,0.20546875,0.36527777777777776,0.5019011406844106,0.6482889733840305,0.47338403041825095,0.23954372623574144,0.009505703422053232,0.2737642585551331,0.7015209125475285,0.27756653992395436,0.6577946768060836,-0.058935361216730035,0.20342205323193915,0.3269961977186312,0.46153846153846156,0.43478260869565216,0.4015748031496063,0.5384615384615384
images/20191123163448.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.3268853081597222,0.2927396797839506,20191123163448000,0.487109375,0.41597222222222224,0.153515625,0.29791666666666666,0.5648854961832062,0.6433566433566433,0.7302798982188295,-1.1445221445221445,0.03816793893129771,-1.0932400932400932,1.0407124681933841,-1.06993006993007,0.5216284987277354,-0.030303030303030304,0.3944020356234097,-1.0,0.5041322314049587,0.4,0.48951048951048953,0.46511627906976744
images/20191123163448.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.32685275607638886,0.2927396797839506,20191123163448000,0.487109375,0.41597222222222224,0.153515625,0.29791666666666666,0.5648854961832062,0.6433566433566433,0.7302798982188295,-1.1445221445221445,0.03816793893129771,-1.0932400932400932,1.0407124681933841,-1.06993006993007,0.5216284987277354,-0.030303030303030304,0.3944020356234097,-1.0,0.5041322314049587,0.4,0.48951048951048953,0.46511627906976744
images/20191123163444.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.3080810546875,0.1638503086419753,20191123163444000,0.4875,0.41875,0.15234375,0.29583333333333334,0.5512820512820513,0.6291079812206573,0.7025641025641025,-1.1854460093896713,0.015384615384615385,-1.1431924882629108,1.0153846153846153,-1.096244131455399,0.5230769230769231,-0.025821596244131457,0.3641025641025641,-1.039906103286385,0.5041322314049587,0.4418604651162791,0.4857142857142857,0.4782608695652174
images/20191123163444.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.30804850260416666,0.16390576774691357,20191123163444000,0.4875,0.41875,0.15234375,0.29583333333333334,0.5512820512820513,0.6291079812206573,0.7025641025641025,-1.1854460093896713,0.015384615384615385,-1.1431924882629108,1.0153846153846153,-1.096244131455399,0.5230769230769231,-0.025821596244131457,0.3641025641025641,-1.039906103286385,0.5041322314049587,0.4418604651162791,0.4857142857142857,0.4782608695652174
images/20191123163451.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.32678629557291666,0.2927324459876543,20191123163451000,0.494140625,0.4625,0.132421875,0.23541666666666666,0.5073746312684366,0.6017699115044248,0.6342182890855457,-1.640117994100295,-0.13274336283185842,-1.5870206489675516,1.0235988200589972,-1.551622418879056,0.6076696165191741,-0.017699115044247787,0.2448377581120944,-1.480825958702065,0.5196850393700787,0.375,0.46715328467153283,0.45
images/20191123163451.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.32678629557291666,0.2927324459876543,20191123163451000,0.494140625,0.4625,0.132421875,0.23541666666666666,0.5073746312684366,0.6017699115044248,0.6342182890855457,-1.640117994100295,-0.13274336283185842,-1.5870206489675516,1.0235988200589972,-1.551622418879056,0.6076696165191741,-0.017699115044247787,0.2448377581120944,-1.480825958702065,0.5196850393700787,0.375,0.46715328467153283,0.45
images/20191123163451.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.32678629557291666,0.2927324459876543,20191123163451000,0.494140625,0.4625,0.132421875,0.23541666666666666,0.5073746312684366,0.6017699115044248,0.6342182890855457,-1.640117994100295,-0.13274336283185842,-1.5870206489675516,1.0235988200589972,-1.551622418879056,0.6076696165191741,-0.017699115044247787,0.2448377581120944,-1.480825958702065,0.5196850393700787,0.375,0.46715328467153283,0.45
images/20191123163451.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.32678629557291666,0.2927324459876543,20191123163451000,0.494140625,0.4625,0.132421875,0.23541666666666666,0.5073746312684366,0.6017699115044248,0.6342182890855457,-1.640117994100295,-0.13274336283185842,-1.5870206489675516,1.0235988200589972,-1.551622418879056,0.6076696165191741,-0.017699115044247787,0.2448377581120944,-1.480825958702065,0.5196850393700787,0.375,0.46715328467153283,0.45
images/20191123163456.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.5997477213541667,0.8314983603395062,20191123163456000,0.442578125,0.027083333333333334,0.23125,0.4111111111111111,0.4831081081081081,0.6300675675675675,0.46452702702702703,0.23817567567567569,0.02195945945945946,0.27871621621621623,0.6841216216216216,0.28547297297297297,0.6199324324324325,-0.05067567567567568,,,0.48507462686567165,0.4666666666666667,,
images/20191123163456.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.5997151692708333,0.8314983603395062,20191123163456000,0.442578125,0.027083333333333334,0.23125,0.4111111111111111,0.4831081081081081,0.6300675675675675,0.46452702702702703,0.23817567567567569,0.02195945945945946,0.27871621621621623,0.6841216216216216,0.28547297297297297,0.6199324324324325,-0.05067567567567568,,,0.48507462686567165,0.4666666666666667,,
images/20191123163457.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.04699571397569444,0.9821180555555555,20191123163457000,0.476953125,0.022222222222222223,0.23671875,0.42083333333333334,0.5726072607260726,0.6204620462046204,0.5198019801980198,0.24422442244224424,0.06435643564356436,0.264026402640264,,,0.5891089108910891,-0.0429042904290429,0.30528052805280526,0.32673267326732675,,,0.5104895104895105,0.48717948717948717
images/20191123163457.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.04699571397569444,0.9821180555555555,20191123163457000,0.476953125,0.022222222222222223,0.23671875,0.42083333333333334,0.5726072607260726,0.6204620462046204,0.5198019801980198,0.24422442244224424,0.06435643564356436,0.264026402640264,,,0.5891089108910891,-0.0429042904290429,0.30528052805280526,0.32673267326732675,,,0.5104895104895105,0.48717948717948717
images/20191123163441.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.39939643012152776,0.8062644675925926,20191123163441000,0.463671875,0.02013888888888889,0.2109375,0.375,0.5444444444444444,0.6481481481481481,0.49444444444444446,0.2351851851851852,0.009259259259259259,0.26481481481481484,0.7425925925925926,0.26851851851851855,0.6611111111111111,-0.05185185185185185,0.2574074074074074,0.31666666666666665,0.5153846153846153,0.3333333333333333,0.48905109489051096,0.4666666666666667
images/20191123163441.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.39936387803819445,0.8062644675925926,20191123163441000,0.463671875,0.02013888888888889,0.2109375,0.375,0.5444444444444444,0.6481481481481481,0.49444444444444446,0.2351851851851852,0.009259259259259259,0.26481481481481484,0.7425925925925926,0.26851851851851855,0.6611111111111111,-0.05185185185185185,0.2574074074074074,0.31666666666666665,0.5153846153846153,0.3333333333333333,0.48905109489051096,0.4666666666666667
images/20191123163440.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.8970757378472223,0.808227237654321,20191123163440000,0.437890625,0.017361111111111112,0.209375,0.3715277777777778,0.5317164179104478,0.6429906542056075,0.5093283582089553,0.22990654205607478,0.04291044776119403,0.2635514018691589,0.7481343283582089,0.27476635514018694,0.6529850746268657,-0.056074766355140186,0.2555970149253731,0.31962616822429907,0.48854961832061067,0.4444444444444444,0.4351145038167939,0.5
images/20191123163440.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.8970757378472223,0.8081693672839506,20191123163440000,0.437890625,0.017361111111111112,0.209375,0.3715277777777778,0.5317164179104478,0.6429906542056075,0.5093283582089553,0.22990654205607478,0.04291044776119403,0.2635514018691589,0.7481343283582089,0.27476635514018694,0.6529850746268657,-0.056074766355140186,0.2555970149253731,0.31962616822429907,0.48854961832061067,0.4444444444444444,0.4351145038167939,0.5
images/20191123163435.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.13788926866319445,0.2912181712962963,20191123163435000,0.48515625,0.41388888888888886,0.15546875,0.3013888888888889,0.5628140703517588,0.6428571428571429,0.7286432160804021,-1.0921658986175116,0.06030150753768844,-1.0414746543778801,1.0452261306532664,-1.032258064516129,0.507537688442211,-0.029953917050691243,0.4020100502512563,-0.9631336405529954,0.5526315789473685,0.3939393939393939,0.5074626865671642,0.4722222222222222
images/20191123163435.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.13785671657986112,0.2912157600308642,20191123163435000,0.48515625,0.41388888888888886,0.15546875,0.3013888888888889,0.5628140703517588,0.6428571428571429,0.7286432160804021,-1.0921658986175116,0.06030150753768844,-1.0414746543778801,1.0452261306532664,-1.032258064516129,0.507537688442211,-0.029953917050691243,0.4020100502512563,-0.9631336405529954,0.5526315789473685,0.3939393939393939,0.5074626865671642,0.4722222222222222
images/20191123163436.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.4290378146701389,0.3449291087962963,20191123163436000,0.4796875,0.018055555555555554,0.209765625,0.3729166666666667,0.5381750465549349,0.633147113594041,0.4655493482309125,0.186219739292365,-0.0074487895716946,0.21601489757914338,0.6964618249534451,0.2383612662942272,0.6461824953445066,-0.054003724394785846,0.21601489757914338,0.2942271880819367,0.5,0.35,0.44776119402985076,0.4883720930232558
images/20191123163437.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.42900526258680555,0.3449291087962963,20191123163437000,0.494140625,0.46111111111111114,0.13359375,0.2375,0.5058479532163743,0.6111111111111112,0.6345029239766082,-1.5730994152046784,-0.11988304093567251,-1.5204678362573099,0.9853801169590644,-1.4912280701754386,0.5847953216374269,-0.02631578947368421,0.22514619883040934,-1.4035087719298245,0.49586776859504134,0.35,0.44029850746268656,0.5
