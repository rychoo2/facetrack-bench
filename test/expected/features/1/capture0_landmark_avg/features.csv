filename,raw_path,landmark_path,rel_target_x,rel_target_y,timestamp,rel_face_x,rel_face_y,rel_face_size_x,rel_face_size_y,rel_pose_x,rel_pose_y,rel_eye_distance_x,rel_eye_distance_y,rel_left_pupil_x,rel_left_pupil_y,rel_right_pupil_x,rel_right_pupil_y
images/20191123163439.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.9621337890625,0.17541714891975307,20191123163439000,0.471484375,0.46041666666666664,0.133203125,0.2361111111111111,0.17008797653958943,-0.29705882352941176,0.592375366568915,-0.04411764705882353,,,,
images/20191123163439.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.9621337890625,0.17541714891975307,20191123163439000,0.471484375,0.46041666666666664,0.133203125,0.2361111111111111,0.17008797653958943,-0.29705882352941176,0.592375366568915,-0.04411764705882353,,,,
images/20191123163438.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.7459716796875,0.2431785300925926,20191123163438000,0.47109375,0.46111111111111114,0.133203125,0.2361111111111111,0.18475073313782991,-0.3058823529411765,0.5777126099706745,-0.047058823529411764,0.48507462686567165,0.4722222222222222,0.45038167938931295,0.4444444444444444
images/20191123163438.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.7459052191840277,0.24323157793209876,20191123163438000,0.47109375,0.46111111111111114,0.133203125,0.2361111111111111,0.18475073313782991,-0.3058823529411765,0.5777126099706745,-0.047058823529411764,0.48507462686567165,0.4722222222222222,0.45038167938931295,0.4444444444444444
images/20191123163459.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.7609388563368056,0.9831090856481481,20191123163459000,0.452734375,0.02013888888888889,0.209375,0.37222222222222223,0.4962686567164179,0.6455223880597015,0.6529850746268657,-0.05970149253731343,0.48031496062992124,0.4230769230769231,0.4251968503937008,0.46153846153846156
images/20191123163459.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.7609388563368056,0.9831090856481481,20191123163459000,0.452734375,0.02013888888888889,0.209375,0.37222222222222223,0.4962686567164179,0.6455223880597015,0.6529850746268657,-0.05970149253731343,0.48031496062992124,0.4230769230769231,0.4251968503937008,0.46153846153846156
images/20191123163458.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.2961222330729167,0.9832947530864198,20191123163458000,0.4765625,0.025694444444444443,0.231640625,0.41180555555555554,0.5379426644182125,0.6188870151770658,0.5868465430016864,-0.050590219224283306,,,0.4744525547445255,0.5
images/20191123163458.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.29605712890625,0.9832368827160494,20191123163458000,0.4765625,0.025694444444444443,0.231640625,0.41180555555555554,0.5379426644182125,0.6188870151770658,0.5868465430016864,-0.050590219224283306,,,0.4744525547445255,0.5
images/20191123163503.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.06577284071180556,0.07994550540123457,20191123163503000,0.49296875,0.4618055555555556,0.133984375,0.23819444444444443,0.5160349854227405,0.5860058309037901,0.5714285714285714,0.01749271137026239,0.5950413223140496,0.15,0.4961832061068702,0.3953488372093023
images/20191123163502.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.06580539279513889,0.07994550540123457,20191123163502000,0.493359375,0.4618055555555556,0.13359375,0.2375,0.5087719298245614,0.5906432748538012,0.5847953216374269,0.014619883040935672,0.5725806451612904,0.23255813953488372,0.5,0.4418604651162791
images/20191123163500.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.9681871202256944,0.9810860339506173,20191123163500000,0.455078125,0.019444444444444445,0.20546875,0.36527777777777776,0.5019011406844106,0.6482889733840305,0.6577946768060836,-0.058935361216730035,,,,
images/20191123163500.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.9680894639756944,0.9810860339506173,20191123163500000,0.455078125,0.019444444444444445,0.20546875,0.36527777777777776,0.5019011406844106,0.6482889733840305,0.6577946768060836,-0.058935361216730035,,,,
images/20191123163448.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.3268853081597222,0.2927396797839506,20191123163448000,0.487109375,0.41597222222222224,0.153515625,0.29791666666666666,0.5648854961832062,0.6433566433566433,0.5216284987277354,-0.030303030303030304,,,0.4825174825174825,0.46511627906976744
images/20191123163448.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.32685275607638886,0.2927396797839506,20191123163448000,0.487109375,0.41597222222222224,0.153515625,0.29791666666666666,0.5648854961832062,0.6433566433566433,0.5216284987277354,-0.030303030303030304,,,0.4825174825174825,0.46511627906976744
images/20191123163444.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.3080810546875,0.1638503086419753,20191123163444000,0.4875,0.41875,0.15234375,0.29583333333333334,0.5512820512820513,0.6291079812206573,0.5230769230769231,-0.025821596244131457,,,,
images/20191123163444.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.30804850260416666,0.16390576774691357,20191123163444000,0.4875,0.41875,0.15234375,0.29583333333333334,0.5512820512820513,0.6291079812206573,0.5230769230769231,-0.025821596244131457,,,,
images/20191123163451.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.32678629557291666,0.2927324459876543,20191123163451000,0.494140625,0.4625,0.132421875,0.23541666666666666,1.2654867256637168,0.37168141592920356,0.5663716814159292,-0.032448377581120944,0.5196850393700787,0.375,0.46715328467153283,0.45
images/20191123163451.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.32678629557291666,0.2927324459876543,20191123163451000,0.494140625,0.4625,0.132421875,0.23541666666666666,1.2654867256637168,0.37168141592920356,0.5663716814159292,-0.032448377581120944,0.5196850393700787,0.375,0.46715328467153283,0.45
images/20191123163451.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.32678629557291666,0.2927324459876543,20191123163451000,0.494140625,0.4625,0.132421875,0.23541666666666666,1.2654867256637168,0.37168141592920356,0.5663716814159292,-0.032448377581120944,0.5196850393700787,0.375,0.46715328467153283,0.45
images/20191123163451.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.32678629557291666,0.2927324459876543,20191123163451000,0.494140625,0.4625,0.132421875,0.23541666666666666,1.2654867256637168,0.37168141592920356,0.5663716814159292,-0.032448377581120944,0.5196850393700787,0.375,0.46715328467153283,0.45
images/20191123163456.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.5997477213541667,0.8314983603395062,20191123163456000,0.442578125,0.027083333333333334,0.23125,0.4111111111111111,0.4831081081081081,0.6300675675675675,0.6199324324324325,-0.05067567567567568,0.47761194029850745,0.43333333333333335,,
images/20191123163456.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.5997151692708333,0.8314983603395062,20191123163456000,0.442578125,0.027083333333333334,0.23125,0.4111111111111111,0.4831081081081081,0.6300675675675675,0.6199324324324325,-0.05067567567567568,0.47761194029850745,0.43333333333333335,,
images/20191123163457.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.04699571397569444,0.9821180555555555,20191123163457000,0.476953125,0.022222222222222223,0.23671875,0.42083333333333334,0.5726072607260726,0.6204620462046204,0.5891089108910891,-0.0429042904290429,,,0.5244755244755245,0.46153846153846156
images/20191123163457.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.04699571397569444,0.9821180555555555,20191123163457000,0.476953125,0.022222222222222223,0.23671875,0.42083333333333334,0.5726072607260726,0.6204620462046204,0.5891089108910891,-0.0429042904290429,,,0.5244755244755245,0.46153846153846156
images/20191123163441.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.39939643012152776,0.8062644675925926,20191123163441000,0.463671875,0.02013888888888889,0.2109375,0.375,0.5444444444444444,0.6481481481481481,0.6611111111111111,-0.05185185185185185,0.5153846153846153,0.3333333333333333,0.48175182481751827,0.4666666666666667
images/20191123163441.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.39936387803819445,0.8062644675925926,20191123163441000,0.463671875,0.02013888888888889,0.2109375,0.375,0.5444444444444444,0.6481481481481481,0.6611111111111111,-0.05185185185185185,0.5153846153846153,0.3333333333333333,0.48175182481751827,0.4666666666666667
images/20191123163440.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.8970757378472223,0.808227237654321,20191123163440000,0.437890625,0.017361111111111112,0.209375,0.3715277777777778,0.5317164179104478,0.6429906542056075,0.6529850746268657,-0.056074766355140186,0.48091603053435117,0.4444444444444444,,
images/20191123163440.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.8970757378472223,0.8081693672839506,20191123163440000,0.437890625,0.017361111111111112,0.209375,0.3715277777777778,0.5317164179104478,0.6429906542056075,0.6529850746268657,-0.056074766355140186,0.48091603053435117,0.4444444444444444,,
images/20191123163435.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.13788926866319445,0.2912181712962963,20191123163435000,0.48515625,0.41388888888888886,0.15546875,0.3013888888888889,0.5628140703517588,0.6428571428571429,0.507537688442211,-0.029953917050691243,,,,
images/20191123163435.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.13785671657986112,0.2912157600308642,20191123163435000,0.48515625,0.41388888888888886,0.15546875,0.3013888888888889,0.5628140703517588,0.6428571428571429,0.507537688442211,-0.029953917050691243,,,,
images/20191123163436.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.4290378146701389,0.3449291087962963,20191123163436000,0.4796875,0.018055555555555554,0.209765625,0.3729166666666667,0.5381750465549349,0.633147113594041,0.6461824953445066,-0.054003724394785846,0.5,0.35,0.44776119402985076,0.4883720930232558
images/20191123163437.jpg,data/raw/capture0/positions.csv,data/landmarks/1/capture0/landmarks.csv,0.42900526258680555,0.3449291087962963,20191123163437000,0.494140625,0.46111111111111114,0.13359375,0.2375,0.15204678362573099,-0.29239766081871343,0.5614035087719298,-0.038011695906432746,0.49586776859504134,0.35,0.44029850746268656,0.5
